README.md

```diff
@@ -14,113 +14,13 @@ This repository serves the website of PyCon TW, Python Conference Taiwan. This p
 - Yarn 1.0+
 - Node.js 8.0+
 
-### Set up a Virtual Environment
+### Set up a Development Environment & Run Server
 
-#### Database - Docker (Optional)
+#### Method 1 : `Quick start in 5 minutes`
+* [ deploy with .sh and docker ](/document/deploy_docker_dev.md)
+#### Method 2 : `Step by step`
+* [ deploy in local environment ](/document/deploy_local_env_dev.md)
```

Comment:
nit: The term `deploy` mostly refers to running the program outside your local machine. Consider changing them (file names & docs).

> ‘Deployment’ is the process of putting a new application, or new version of an application, onto a prepared application server. (quoted from [this article](https://codefol.io/posts/deployment-versus-provisioning-versus-orchestration/))

```suggestion
#### Method 1 : `Quick Start`
* [ Run with docker-compose & shell scripts ](/document/deploy_docker_dev.md)
#### Method 2 : `Step by step`
* [ Launch on your local runtime ](/document/deploy_local_env_dev.md)
```

Reply:
Done!